Microsoft.Toolkit.Uwp.UI.Controls.Layout/ListDetailsView/ListDetailsView.xaml

```diff
@@ -13,92 +14,122 @@
-                                  BorderBrush="{TemplateBinding BorderBrush}"
-                                  BorderThickness="0,0,1,0">
-                                <Grid.RowDefinitions>
-                                    <RowDefinition Height="Auto" />
-                                    <RowDefinition Height="*" />
-                                    <RowDefinition Height="Auto" />
-                                </Grid.RowDefinitions>
-                                <ContentPresenter x:Name="HeaderContentPresenter"
-                                                  Margin="12,0"
-                                                  x:DeferLoadStrategy="Lazy"
-                                                  Content="{TemplateBinding ListHeader}"
-                                                  ContentTemplate="{TemplateBinding ListHeaderTemplate}"
-                                                  Visibility="Collapsed" />
-                                <ListView x:Name="List"
-                                          Grid.Row="1"
-                                          IsTabStop="False"
-                                          ItemContainerStyle="{TemplateBinding ItemContainerStyle}"
-                                          ItemContainerStyleSelector="{TemplateBinding ItemContainerStyleSelector}"
-                                          ItemTemplate="{TemplateBinding ItemTemplate}"
-                                          ItemTemplateSelector="{TemplateBinding ItemTemplateSelector}"
-                                          ItemsSource="{TemplateBinding ItemsSource}"
-                                          SelectedItem="{Binding SelectedItem, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}" />
-                                <Grid x:Name="ListCommandBarPanel" Grid.Row="2"></Grid>
-                            </Grid>
-                            <Grid x:Name="DetailsPanel"
-                                  Grid.Column="1">
-                                <ContentPresenter x:Name="NoSelectionPresenter"
-                                                  Content="{TemplateBinding NoSelectionContent}"
-                                                  ContentTemplate="{TemplateBinding NoSelectionContentTemplate}" />
-                                <Grid x:Name="SelectionDetailsPanel">
+                        <muxc:TwoPaneView x:Name="RootPanel"
+                                          MinWideModeWidth="{TemplateBinding CompactModeThresholdWidth}"
+                                          Pane1Length="{TemplateBinding ListPaneWidth}"
+                                          PanePriority="Pane1"
+                                          TallModeConfiguration="SinglePane"
+                                          WideModeConfiguration="LeftRight">
+                            <muxc:TwoPaneView.Pane1>
+                                <Grid x:Name="MasterPanel"
```

Comment:
Shouldn't this be ListPanel to match new naming

Reply:
Yes. Done.